app/presenters/manual_presenter.rb

```diff
@@ -14,6 +14,10 @@ def beta?
     true
   end
 
+  def slug_for_search
+    url[1..-1]
```

Comment:
Adding this information into the content item would be a nicer way to do this.

Since updating the publishing tool to include that and republishing all the manuals is somewhat outside the scope of this work, and shouldn't block getting this live, I'm going to merge this now and leave that tidy up for another time.